docs/plans/02-key-and-harmony-tests.md

```
@@ -0,0 +1,66 @@
+# Test Plan: Key & Harmony Detection
+
+## Scope
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)